### ISSUE TITLE
import page layout is slightly broken for local imports

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#3324]
### Observed behavior

![image](https://user-images.githubusercontent.com/2367265/37182477-6187c1bc-22e6-11e8-964a-7482d733a601.png)

### Expected behavior

We should avoid floats if possible to prevent this

### User-facing consequences

app looks wonky, might hinder usage of filters

### Steps to reproduce

import from local drive on a small window

### Context

k0.8

----

reported by @jamalex on his tiny ubuntu device

**Comment by DXCanas** [comment]
What's the ideal behavior here? wrap? or take up as much width as necessary until it's forced to wrap (edge of screen)?

**Comment by indirectlylit** [comment]
This is one of the [non-standard layouts](https://docs.google.com/document/d/1dpKGD_nMzYmqa82EmLdLGMZTIodJ6MJFLS63ijSr7UM/edit#) so there's no one correct answer. As I mentioned in that doc, I think it's best to define these layouts in terms of [proportional grids](https://purecss.io/grids/) and our [standard breakpoints](https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/mixins/responsive-window.js#L26). Note in that file that [Material design gives guidance](https://material.io/guidelines/layout/responsive-ui.html#responsive-ui-grid) on details like what column granularity to use at different breakpoint sizes, and what gutter widths to use.

Basically, blocks are specified as taking up some integer multiple of a column width. Within a block, text might wrap as necessary.

Just for sake of argument, I would expect a responsive layout spec to look like something this:

-----

## > 600 px

8 col granularity:

```
|       |       |       |       |       |       |       |       |
```

layout:

```
| title                         |       | lang  | search        |
```

widths:

* title - 4 x 1/8; spacer - 1 x 1/8; lang - 1 x 1/8; search - 2 x 1/8


## 480 - 600 px

4 col granularity:

```
|               |               |               |               |
```

layout:

```
| title                                                         |
| lang          | search                                        |
```

widths:

* title - 4 x 1/4
* lang - 1 x 1/4; search - 3 x 1/4

## < 480 px

4 col granularity:

```
|               |               |               |               |
```

layout:

```
| title                                                         |
| lang                                          |               |
| search                                                        |
```

widths:

* title - 4 x 1/4
* lang - 3 x 1/4;  spacer - 1 x 1/4
* search - 4 x 1/4

**Comment by indirectlylit** [comment]
fyi, take a look at the [export data](http://kolibribeta.learningequality.org/facility/#/data) page for an example of how this [can be coded up](https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/facility_management/assets/src/views/data-page/index.vue#L75).

**Comment by indirectlylit** [comment]
fixed in #4039
